packages/google_sign_in/google_sign_in_web/lib/src/gis_client.dart

```diff
@@ -291,7 +293,12 @@ class GisSdkClient {
   /// Requests a server auth code per:
   /// https://developers.google.com/identity/oauth2/web/guides/use-code-model#initialize_a_code_client
   Future<String?> requestServerAuthCode() async {
-    _codeClient.requestCode();
+    assert(_codeClient != null,
+        'CodeClient not initialized correctly. Ensure the `scopes` list passed to `init()` or `initWithParams()` is not empty!');
```

Comment:
Can we have a TODO with an issue link here to track re-creating `_codeClient` when adding scopes?